proto/hypertrace/agent/config/v1/config.proto

```diff
@@ -195,5 +195,21 @@ message Telemetry {
 
     // Whether to capture metrics or not. The metrics will be otel go metrics.
     // See https://github.com/open-telemetry/opentelemetry-go/tree/main/metric
-    google.protobuf.BoolValue metrics_enabled = 2;
+    google.protobuf.BoolValue metrics_enabled = 2 [deprecated = true];
 }
+
+// Metrics has the config to enable OTEl and System metrics
+message Metrics {
+    // Whether to capture metrics or not. The metrics will be otel go metrics.
+    // See https://github.com/open-telemetry/opentelemetry-go/tree/main/metric
+    google.protobuf.BoolValue metrics_enabled = 1;
```

Comment:
just `enabled` would be fine

Reply:
keeping just `enabled`, is creating env var as `HT_ENABLED`, making it like this creates `HT_METRICS_ENABLED`, so named it like this. what do you suggest @varkey98 ?

Reply:
@thugrock7 resolve [this](https://github.com/hypertrace/agent-config/pull/163#discussion_r1677519678) comment and it'll get fixed. It's happening because this message is a dangling one and not referred by anything else